### PR TITLE
fix(wizard): client side validation for java package groupid

### PR DESCRIPTION
**Pull request by corinnekrych**
fixes https://github.com/openshiftio/openshift.io/issues/1158

### CODE REVIEW

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2225-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2225-fabric8-ui.badger.fabric8.io

File: src/app/space/forge-wizard/quickstart-pages/step2/project-info-step.component.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ export class ProjectInfoStepComponent implements OnInit {
       Validators.minLength(this.MIN_LENGTH),
       Validators.maxLength(this.MAX_LENGTH)]);
 
+    this.form.controls['groupId'].setValidators(Validators.pattern(/^[a-z][a-z_0-9\.]*[a-z_0-9]$/));
```
Comment (joshuawilson) [inline src/app/space/forge-wizard/quickstart-pages/step2/project-info-step.component.ts]: This looks something that could do with a unit test.

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2225-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2225-fabric8-ui.badger.fabric8.io

**Comment by corinnekrych** [comment]
@joshuawilson I've added component tests as suggested.

**Pull request merged**